cli/pkg/pgconn/connection.go

```diff
@@ -135,23 +135,46 @@ func ParseDBURI(dbURI string) (*pgxpool.Config, error) {
 	return db, nil
 }
 
-func constructURI(user, password, host string, port int, dbname, sslmode string, connectTimeout int) string {
+func ConstructURI(connDetails pgconn.Config, sslmode string) string {
 	c := new(url.URL)
 	c.Scheme = "postgres"
-	c.Host = fmt.Sprintf("%s:%d", host, port)
-	if password != "" {
-		c.User = url.UserPassword(user, password)
+	c.Host = fmt.Sprintf("%s:%d", connDetails.Host, connDetails.Port)
+	if connDetails.Password != "" {
+		c.User = url.UserPassword(connDetails.User, connDetails.Password)
 	} else {
-		c.User = url.User(user)
+		c.User = url.User(connDetails.User)
 	}
-	c.Path = dbname
+	c.Path = connDetails.Database
 	q := c.Query()
 	if sslmode != "" {
 		q.Set("sslmode", sslmode)
 	}
-	if connectTimeout != 0 {
-		q.Set("connect_timeout", strconv.Itoa(connectTimeout))
+
+	if connDetails.ConnectTimeout != 0 {
+		q.Set("connect_timeout", strconv.Itoa(int(connDetails.ConnectTimeout.Seconds())))
 	}
 	c.RawQuery = q.Encode()
 	return c.String()
 }
+
+func getPromscaleSecretName(releaseName, namespace string) (string, error) {
+	helmClient := helm.NewClient(namespace)
+	defer helmClient.Close()
+	eS, err := helmClient.ExportValuesFieldFromRelease(releaseName, []string{"promscale", "connectionSecretName"})
+	if err != nil {
+		return "", err
+	}
+	secretProvided, ok := eS.(string)
+	if !ok {
+		return "", fmt.Errorf("promscale.connectionSecretName is not a string")
+	}
+
+	var secretName string
+	if secretProvided == "" {
+		secretName = releaseName + "-promscale"
+	} else {
+		secretName = secretProvided
+	}
+
+	return secretName, nil
+}
```

Comment:
The same function is added in `cli/cmd/common/common.go`, could you reuse that one here?